fava/ext/portfolio_list/templates/portfolio_list.html

```diff
@@ -0,0 +1,13 @@
+{% extends "_layout.html" %}
+{% import "_query_table.html" as querytable with context %}
+{% set active_page = 'PortfolioList' %}
```

Comment:
Likewise, the extension should really only provide what's in content block

Reply:
Done, though i still import query_table for the sample extension